infra/nightly.sh

```diff
@@ -62,6 +62,9 @@ fi
 # Generate latex after running the profiler (depends on profile.json)
 ./infra/generate_line_counts.py "$NIGHTLY_DIR"
 
+# Generate CFGs for LLVM after running the profiler
+./infra/generate_cfgs.py "$NIGHTLY_DIR/data/llvm"
```

Comment:
Might be better for the `profile.py` file to call `generate_cfgs` directly- less bash communication is better

Reply:
I kind of disagree? I think the separation is good- `profile.py` is in charge of running all the benchmarks and stuff, `generate_cfgs` is in charge of reading that data in and writing pngs out.